content/standards/external-standards.html.md

```diff
@@ -0,0 +1,10 @@
+---
+title: External Standards Catalogues
+
+---
+
+# External Standards Catalogues
+
+- [Criminal Justice System Exchange Data Standards Catalogue](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/360582/cjs-data-standards-catalogue-5.0.pdf)
```

Comment:
Is there only a PDF, no other versions?

Reply:
I can't see another one :( there is this page but it just links back to the PDF which has the actual content https://www.gov.uk/guidance/criminal-justice-system-data-standards-forum-guidance#data-standards-catalogue

Reply:
I've only seen the PDF...